Microsoft.Toolkit.Uwp.UI.Animations/Extensions/AnimationExtensions.cs

```diff
@@ -23,7 +23,6 @@ namespace Microsoft.Toolkit.Uwp.UI.Animations
     /// </summary>
     public static partial class AnimationExtensions
     {
-        private static readonly EasingFunctionBase _defaultStoryboardEasingFunction = new CubicEase();
```

Comment:
What do you think about having a public default easing function that can be the default unless otherwise specified?